Auth0.podspec

```diff
@@ -16,11 +16,30 @@ web_auth_files = [
   'Auth0/SilentSafariViewController.swift',
   'Auth0/NativeAuth.swift',
   'Auth0/AuthProvider.swift',
-  'Auth0/CredentialsManager.swift',
-  'Auth0/CredentialsManagerError.swift',
   'Auth0/BioAuthentication.swift'
 ]
 
+watchos_exclude_files = [
```

Comment:
this looks like `web_auth_files` plus 2 entries. Is there a nicer way to define this array? i.e. joining the already defined base array.

Reply:
Sadly not, i've tried joining arrays etc it will just throw out an error in cocoapods when linting the lib.